src/08-accessing.js

```diff
@@ -171,7 +193,10 @@ function getFullAddress(person) {}
       numbers: [7185550921, 7185558611],
     };
  */
-function getFlatObject(person) {}
+function getFlatObject(person) {
+  let newPerson = Object.create(person)
```

Comment:
I love the ingenuity here!  ` Object.create() ` is a great method.  Can you think of another way to do this?